data/hai ships.txt

```diff
@@ -36,6 +36,7 @@ ship "Aphid"
 		"Hai Fissure Batteries"
 		"Hai Corundum Regenerator"
 		"Hai Williwaw Cooling"
+		"Pulse Rifle"
```

Comment:
Drawing on feedback given for #3427, the variants of a given model should have different numbers of rifles.

I think more aggressive variants (e.g. one with an offensive turret vs defensive outfits only) should probably have more rifles than their counterparts, since these ships are more likely to go on the attack and become disabled vs those that are conservative and in a supporting role.

(i.e. "Aphid" -> 2 or 3 Pulse Rifles, while "Aphid (Anti-Misisle)" staying at 1 makes sense)

Reply:
okay I will work on that